mappings/markdown/Zeek/v.1.3.0/conn_log/README.md

```diff
@@ -0,0 +1,82 @@
+| ----------------------------- | --------------- | --------------------------------------------------------------------------------------- | ----------------------- |
+| `time`                        | `ts`            | Time when the connection began.                                                         | Convert to epoch value. <br>Type is Integer. |
+| `start_time`                  | `ts`            | Time when the connection began.                                                         | Convert to epoch value. <br>Type is Integer. |
+| `metadata.logged_time`        | `_write_ts`     | Timestamp indicating when the log entry was written to disk.                            | Convert to epoch value. <br>Type is Integer. |
+| `metadata.loggers[].name`     | `_system_name`  | Name of the system or logging subsystem generating the log entry.                       |                         |
+| `metadata.log_name`           | `_path`         | Log name.                                                                               |                         |
+| `metadata.uid`                | `uid`           | Unique ID for the connection.                                                           |                         |
+| `src_endpoint.ip`             | `id.orig_h`     | The originator’s IP address.                                                            |                         |
+| `src_endpoint.port`           | `id.orig_p`     | The originator’s port number.                                                           | Type is Integer.        |
+| `src_endpoint.location.country` | `orig_cc`     | Country code for GeoIP lookup of the originating IP address.                            |                         |
+| `src_endpoint.mac`            | `orig_l2_addr`  | Link-layer address of the originator, if available.                                     |                         |
+| `dst_endpoint.ip`             | `id.resp_h`     | The responder’s IP address.                                                             |                         |
+| `dst_endpoint.port`           | `id.resp_p`     | The responder’s port number.                                                            | Type is Integer.        |
+| `dst_endpoint.location.country` | `resp_cc`     | Country code for GeoIP lookup of the responding IP address.                             |                         |
+| `dst_endpoint.mac`            | `resp_l2_addr`  | Link-layer address of the responder, if available.                                      |                         |
+| `app_name`                    | `service`       | An identification of an application protocol being sent over the connection.            |                         |
+| `connection_info.community_uid` | `community_id`| The community ID.                                                                       | Unmapped in v1.3.0, mapped in v1.4.0-dev |
```

Comment:
It looks like this caused a validation error with `1.3.0`.

```
  "errors": [
    {
      "error": "attribute_unknown",
      "message": "Unknown attribute at \"connection_info.community_uid\"; attribute \"community_uid\" is not defined in object \"network_connection_info\".",
      "attribute": "community_uid",
      "object_name": "network_connection_info",
      "attribute_path": "connection_info.community_uid"
    }
  ],
  "error_count": 1,
```

I think the `community_uid` attribute may have been introduced in `1.4.0-dev`

Reply:
Yeah, I noted that in the notes column for it: 
> | Unmapped in v1.3.0, mapped in v1.4.0-dev |
I intend to update everything to v1.4.0 once it's out of dev

Reply:
Have you considered adding two entries in this PR, one that's valid for `1.3.0` and another that's prepped for `1.4.0`? It might lessen the hassle of having to do a separate PR to clean things up.

Reply:
Maybe I'm not understanding how to do that?   Oh, you mean like duplicate it as a separate folder that has just that one change?

Reply:
Yes, a folder for the 1.3.0 mapping and the 1.4.0 mapping. That way there is a clean spec for this event across both 1.3.0 and 1.4.0 without any unknown attributes.